qa/standalone/ceph-helpers.sh

```diff
@@ -152,11 +152,15 @@ function test_setup() {
 #
 function teardown() {
     local dir=$1
+    local dumplogs=$2
     kill_daemons $dir KILL
     if [ `uname` != FreeBSD ] \
         && [ $(stat -f -c '%T' .) == "btrfs" ]; then
         __teardown_btrfs $dir
     fi
+    if [ "$dumplogs" = "1" ]; then
+        display_logs $dir
+    fi
```

Comment:
@dzafman 
Can we think of something else here? Or add a flag to suppress this?
Dumping all the logs of ceph-helper is killing me, and my browser.
I also don't like it in the Jenkins runs, but have not found a better replacement for it.
Perhaps there is not...

Not run-standalone is mostly run from the CLI as would I expect?
Or is this specifically for the teuthology runs?

If you crank up any logging you might en up with over 450Mb of logging, and my firefox really does not like that.